docs/source/apps/projinfo.rst

```diff
@@ -44,6 +44,7 @@ Synopsis
       (*added in 6.2*)
     - a OGC URN combining references for concatenated operations
       (e.g. "urn:ogc:def:coordinateOperation,coordinateOperation:EPSG::3895,coordinateOperation:EPSG::1618")
+    - a PROJJSON string. The jsonschema is at https://github.com/OSGeo/proj/blob/master/data/crsjson.schema.json (*added in 6.2*)
```

Comment:
In the schema file this is said to be located at `https://proj.org/crsjson.schema.json`. I would prefer to use the same location here too. Having a direct link to master on github is a potential risk if the schema is updated before a new release is published.